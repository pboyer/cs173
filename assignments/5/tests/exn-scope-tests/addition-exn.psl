try
    +(2, "not a num")
catch e in {
  print(e.type);
  print("\n");
  print(e.message);
}
