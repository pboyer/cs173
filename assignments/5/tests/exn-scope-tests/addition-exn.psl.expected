PSL
Bad arguments to +Bad arguments to +
